js/supermarkets.js

```diff
@@ -186,7 +182,7 @@ function clearLocations() {
             markers = [];
 }
 
-/**ATTENTION! CODE belo is just temporary stored, it will be deleted */
+/**ATTENTION! CODE below is just temporary stored, it will be deleted */
```

Comment:
When this code will be deleted? If you plan to merge this pr into master then you need to delete all commented code stored here temporarily.